crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -473,3 +477,41 @@ pub struct IatapayAccessTokenErrorResponse {
     pub error: String,
     pub path: String,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IatapayPaymentWebhookBody {
+    pub status: IatapayWebhookStatus,
+    pub iata_payment_id: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IatapayRefundWebhookBody {
+    pub status: IatapayWebhookStatus,
```

Comment:
incase of refund failure error_message, error_code and error_reason should be captured